onnxruntime/wasm/api.h

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <emscripten.h>
+
+#include <stddef.h>
+
+namespace Ort {
+struct Session;
+}
+typedef Ort::Session* ort_session_handle_t;
+
+struct OrtValue;
+typedef OrtValue* ort_tensor_handle_t;
+
+extern "C" {
+
+/**
+ * perform global initialization. should be called only once.
+ */
+void EMSCRIPTEN_KEEPALIVE ort_init();
+
+/**
+ * create an instance of ORT session.
+ * @param data a pointer to a buffer that contains the model.
+ * @param data_length the size of the buffer in bytes.
+ * @returns a handle of the ORT session.
+ */
+ort_session_handle_t EMSCRIPTEN_KEEPALIVE ort_create_session(void* data, size_t data_length);
```

Comment:
Function names in this file don't conform to the conventions.

Reply:
Do you mean the PascalCase function name with "Ort" as prefix?

Those C-Style functions are not consumed by any C++ users; instead it is consumed by emcc to generate the .wasm file, which will be used by Javascript code. Maybe we should align with javascript convention? (usually a function is a camelCase)